assets/search/modmail/result.ejs

```diff
@@ -0,0 +1,6 @@
+<h4 class="list-group-item-heading">
+    {{result.subject}} <em>by /u/{{result.author}}</em>
+</h4>
+<p class="list-group-iitem-text">
+    {{result.body}} <em>{{result.created_utc * 1000 | date:'yyyy-MM-dd HH:mm:ss'}}</em>
```

Comment:
I know you don't like this in the title, but maybe stick it bolded at the top of the p?